.github/workflows/_e2e.yml

```
@@ -33,7 +34,7 @@ jobs:
   # Run e2e test against localnet built on the develop branch
   localnet:
     name: Localnet
-    if: inputs.isExplorer || inputs.isTypescriptSDK || inputs.isWallet || inputs.isRust || github.ref_name == 'develop'
+    if: inputs.isExplorer || inputs.isTypescriptSDK || inputs.isWallet || github.ref_name == 'develop'
```

```suggestion
    if: inputs.isExplorer || inputs.isTypescriptSDK || inputs.isWallet
```
@DaughterOfMars should we also do this?

---

Or would it prevent it from running from the nightly workflow?

---

It would prevent it from running in nightly, yes

---

But does leaving it means it will run on pushes to develop?

---

I guess not because it won't be called by the hierarchy?

---

Yes, exactly.